Tasks/ANT/task.json

```diff
@@ -16,7 +16,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 125,
+        "Minor": 126,
         "Patch": 0
```

Comment:
Do we want to use 125.1 since these will be hotfixed in 125?

Reply:
Would that make things easier?  (I figured I'd have to go line-by-line anyway, and figured at least the sprint boundaries would be useful.)

Reply:
Its fine to use 126 as long as any other changes we make in 126 update the patch version